### PR TITLE
Feature Admin activates and Deactivates users

### PR DESCRIPTION
**Pull request by sabin18**
#### What does this PR do?
- update deactivate user endpoint and activate user end point.
#### Description of Task to be completed?
- change from POST method to PATCH method on routes
#### How should this be manually tested?
- clone the repository : git clone https://github.com/alainmateso/iFind.git
- checkout to branch Ft-activate-168535853
#### Any background context you want to provide?
`N/A`
#### What are the relevant pivotal tracker stories?
[#168535853](https://www.pivotaltracker.com/story/show/168535853)
#### Screenshots (if appropriate)
`N/A`
#### Questions:
`N/A`

### CODE REVIEW

File: __test__/__admin__/adminTest.spec.js [hunk 1]
```diff
@@ -100,7 +100,7 @@ it('Should not create category twice', (done) => {
 });
 it('Should activate user', (done) => {
     chai.request(app)
-      .post('/api/v1/admin/activate/1')
```
Comment (ngireric123) [inline __test__/__admin__/adminTest.spec.js:103]: it is ok now

**Review by ngireric123 — APPROVED** [review]

**Review by Cheza-Dzabala — APPROVED** [review]
cool

**Review by alainmateso — COMMENTED** [review]
Looks okay

**Comment by nignanthomas** [comment]
👍

**Pull request merged**